source/_patterns/02-components/cards/card.scss

```diff
@@ -1,61 +1,40 @@
+@use "@db-ui/base/build/scss/init" as *;
 @use "@db-ui/base/build/scss/variables" as *;
+@use "@db-ui/base/build/scss/variables.global" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
-@import "cards.variables";
+@use "../../00-base/color-variants" as *;
 
-.cmp-card {
-	background-color: #fff;
+.db-card {
+	@extend %focus-placeholder;
+	@extend %color-variants;
+
+	background-color: var(
+		--db-current-background-color,
+		$db-colors-neutral-bg-0-enabled
+	);
```

Comment:
why don't we extend a tonality for this one instead of setting a `background-color` directly?

Reply:
We can use any color-container neutral-1...6 or success, primary and so on. 
And those containers overwrite the --db-current-background-color for all children.
Therefore we need to use it like this

Reply:
got it, seems I misinterpreted its meaning previously. Thanks.